.gitattributes

```diff
@@ -0,0 +1 @@
+*.tiff filter=lfs diff=lfs merge=lfs -text
```

Comment:
Just wondering whether a catch-all `*.tiff` is risky here? I think that we should be more explicit either here or in the `.gitignore`. Not sure if you've encountered problems with Git LFS before, but I have, and it can very quickly consume all your free quota - I've never paid for it though.